lab-doug/gulpfile.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const nodemon = require('nodemon');
+const mocha = require('gulp-mocha');
+
+const paths = ['*.js', 'lib/*.js', 'model/*.js', 'route/*.js', 'test/*.js'];
+
+
+gulp.task('eslint', function(){
+  gulp.src(paths)
+  .pipe(eslint())
+  .pipe(eslint.format())
+  .pipe(eslint.failAfterError());
+});
+
+gulp.task('test', () => {
+  return gulp.src(['./test/receipt-router-test.js', './test/repair-router-test.js'], {read: false})
```

Comment:
I might do something like './test/*-test.js' so that it's general for even when you add new test files.